### DISCUSSION TITLE
Avoid ellipsis from title header display

### DISCUSSION THREAD

**Discussion by pcbal** [payloadcms/payload#2151] [Feature Requests & Ideas]
How to remove ellipsis from title header and show full title when useAsTitle for long entries and enable line break and overflow visible.

**Comment by DanRibbens** [comment]
This is possible with custom css: https://payloadcms.com/docs/admin/customizing-css

I would love to see before and after screenshots once you get it.

**Reply by pcbal** [reply]
Thanks for the reply and your care. But this I tried and failed. CSS override did not help here. Attached are the before and after.
Before override
![06-03-2023-10:24:56](https://user-images.githubusercontent.com/40548855/223023018-3dcb7c00-2f5d-48e7-be63-8abd15693045.png)
After css override 
&__header {
    h1 {
      white-space: nowrap;
      overflow: visible;
      text-overflow: auto;
      line-height: 1.25;
      display: block;
    }
  }
![06-03-2023-10:22:59](https://user-images.githubusercontent.com/40548855/223023050-96bb83b5-72f8-484c-a2f6-f437e756a3d9.png)
